### PR TITLE
ADEN-8302 Switch Beachfront to via-DFP integration

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/ad-bidders/prebid/adapters/beachfront.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ export class Beachfront extends BaseAdapter {
 			code,
 			mediaTypes: {
 				video: {
+					context: 'instream',
```
Comment (fraszczakszymon) [inline src/ad-bidders/prebid/adapters/beachfront.js]: Shouldn't we use 'outstream'? Shouldn't it be configurable per slot?

Comment (dmnsobczak) [reply src/ad-bidders/prebid/adapters/beachfront.js]: If you use `outstream` then Prebid won't use VAST cache and will try to return bid + renderer/player. We ask for `instream` and then we do our custom in-DFP magic.

File: src/ad-services/confiant/index.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ class Confiant {
 			mapping,
 			activation,
 			callback: (...args) => {
-				console.log('w00t one more bad ad nixed.', args);
```
Comment (nandy-andy) [inline src/ad-services/confiant/index.js]: 😆

Comment (dmnsobczak) [reply src/ad-services/confiant/index.js]: It was messing with my linter output :x

**Pull request merged**